### ISSUE TITLE
Clear logs

### ISSUE THREAD

**Issue by Tam** [jesseduffield/lazydocker#403]
It'd be really useful to be able to clear the logs view and only show new items as they come in.

This is possible when viewing the logs independently (i.e. via the `m` keybind) and using the clear command to clear the terminal, but I often find myself jumping between containers to check the logs, so being able to do this within the lazydocker context would be handy!

**Comment by jesseduffield** [comment]
I agree this would be a great feature. It might also be quite easy i.e. just press a key and then clear the main view. I'm gonna put a good-first-issue label on this. If you wanna take a stab at this @Tam lemme know and I can point you to the right places in the code to make the changes

**Comment by Tam** [comment]
It's been a minute since I last used Go @jesseduffield, but sure if you can point me in the right direction I'll try to find time soon to give it a shot!

**Comment by jesseduffield** [comment]
Awesome :)

So you'll want to add a keybinding handler in `pkg/gui/main_panel.go` called `handleClearMain` 
```go
func (gui *Gui) handleClearMain() error {
	if gui.popupPanelFocused() {
		return nil
	}

    gui.Views.Main.Clear()
	return nil
}
```
and then you'll want to reference it from `pkg/gui/keybindings.go` like so:
```go
		{
			ViewName: "main",
			Key:      gocui.KeyCtrlL,
			Modifier: gocui.ModNone,
			Handler:  wrappedHandler(gui.handleClearMain),
			Description: gui.Tr.ClearMain,
		},
```

You'll also need to add the keybinding's help text to `pkg/i18n/english.go` (that's what the `gui.Tr.ClearMain` thing refers to).

I'd see how that goes. We'll likely need to tweak it but that's a good starting spot

**Comment by 26tanishabanik** [comment]
@jesseduffield, is anyone actively working on this issue right now?

**Comment by jesseduffield** [comment]
Did you get a chance to start on this @Tam ? If you want to give it a go @26tanishabanik I'm happy to assign you

**Comment by 26tanishabanik** [comment]
> Did you get a chance to start on this @Tam ? If you want to give it a go @26tanishabanik I'm happy to assign you

I can give it a try @jesseduffield

**Comment by Tam** [comment]
@jesseduffield it’s probably gonna be a while before I have time, happy to let @26tanishabanik take the reins 👍

**Comment by 26tanishabanik** [comment]
> ```go
> ClearMain
> ```

@jesseduffield , `KeyCtrlL` , means key 'L'?

**Comment by mark2185** [comment]
I'd guess it's `ctrl+L`.

**Comment by 26tanishabanik** [comment]
> I'd guess it's `ctrl+L`.

Thanks @mark2185

**Comment by 26tanishabanik** [comment]
@jesseduffield , `ctrl+L` is working, I am able to clear the logs, after moving my cursor to the logs tab

**Comment by drusellers** [comment]
I would love a way to trim the current logs - so that I don't have to wait for it to page through all of the non-sense old logs too.

**Comment by hgkdzbf6** [comment]
I've implemented the clear logs functionality and submitted PR #772. The implementation adds Ctrl+L keybinding to clear the main panel's content.

**Comment by vigneshakaviki** [comment]
Claiming this. Working on clear-logs shortcut in lazydocker logs view now.
